Source/WebCore/layout/formattingContexts/inline/TextOnlySimpleLineBuilder.cpp

```diff
@@ -202,6 +202,10 @@ InlineItemPosition TextOnlySimpleLineBuilder::placeInlineTextContent(const Rende
 
     auto nextItemIndex = layoutRange.startIndex();
     auto isAtSoftWrapOpportunityOrContentEnd = [&](auto& inlineTextItem) {
+        // In minimum content mode, treat every character boundary as a wrap opportunity
+        // to ensure we compute the true minimum width (widest single character/word).
```

Comment:
This is not how minimum content width is supposed to work. Could you add a reduced test case that triggers  the ASSERT?